flutter-idea/src/io/flutter/pub/PubRoot.java

```diff
@@ -82,13 +82,13 @@ public static PubRoot forEventWithRefresh(@NotNull final AnActionEvent event) {
       }
     }
 
-    final Project project = event.getData(CommonDataKeys.PROJECT);
-    if (project != null) {
-      final List<PubRoot> roots = PubRoots.forProject(project);
-      if (!roots.isEmpty()) {
-        return roots.get(0);
-      }
-    }
+    //final Project project = event.getData(CommonDataKeys.PROJECT);
```

Comment:
Can you add a comment about why this is commented out?

Reply:
Oops, I forgot to delete those lines. Fixed.